doc/data-files-description.md

```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
+|`fusion_summary_ependymoma_foi.tsv` | Analysis file | [`fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of ependymal tumor fusions of interest |
+|`fusion_summary_ewings_foi.tsv` | Analysis file | [`fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary) | Summary file for presence of Ewing's sarcoma fusions of interest |
+|`gencode.v27.primary_assembly.annotation.gtf.gz` | Reference file | GENCODE v27 | hg38 gene annotation on primary assembly (reference chromosomes and scaffolds) |
+|`GRCh38.primary_assembly.genome.fa.gz` | Reference Genome file | GENCODE v27 | hg38 primary assembly genome sequence FASTA file |
+|`independent-specimens.wgs.primary-plus.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS sample, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgs.primary.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS samples, primary only |
+|`independent-specimens.wgswxs.primary-plus.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary + non-primary when no primary sample is available |
+|`independent-specimens.wgswxs.primary.tsv` | Analysis file | [`independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) | Independent specimens list for WGS and WXS samples, primary only |
+|`intersect_cds_lancet.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions and Lancet's WXS regions |
+|`intersect_cds_lancet_strelka_mutect_WGS.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Lancet, Strelka2, Mutect2 regions |
+|`intersect_strelka_mutect_WGS.bed` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Strelka2 and Mutect2 regions called |
+|`pbta-cnv-cnvkit-gistic.zip` | PBTA data file | [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/bash/run-gistic.sh) | Somatic CNV - GISTIC 2.0 output using `pbta-cnv-cnvkit.seg` file input (WGS samples only) |
```

Comment:
dead link! Maybe this? https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh